src/frontend/src/pages/index.js

```diff
@@ -124,18 +122,17 @@ export default function IndexPage() {
       }
     }
     setPostsInfo();
-  }, []);
+  }, [getPostsCount, currentNumPosts]);
```

Comment:
Does this not need `initNumPosts` too?  Also, why do we call `getPostsCount()` twice on 117 and 118, and not simply re-use the value?

Reply:
We don't care about whatever initNumPosts is going to be in this case. Both currentNumPosts and initNumPosts will be overwritten. I only want this side-effect happening whenever a currentNumPosts value changes